src/modules/dashboard/components/ColonyMembers/ColonyMembers.tsx

```diff
@@ -108,6 +152,19 @@ const ColonyMembers = () => {
     });
   }, [openPermissionManagementDialog, colonyData, isVotingExtensionEnabled]);
 
+  const selectedDomain = colonyData?.processedColony.domains.find(
+    ({ ethDomainId }) => ethDomainId === selectedDomainId,
+  );
```

Comment:
I think we need to add a check to `processedColony`. When navigating to the Members page I am getting null for the value, crashing the DApp.

![Screenshot 2022-05-16 114323](https://user-images.githubusercontent.com/33682027/168565212-59841d28-df63-4bb9-a8a7-4633df5f8caa.png)